wordpress-mobile/WordPress-Android
# Sharing - manage connections

Fixes #4070 

To test:
Toggle all settings in manage connection and verify that it reflects on the web or iOS

Needs review: @nbradbury
